### PR TITLE
Streamline DTWF tests and update test requirements

### LINKED ISSUE #767: DTWF statistical tests falling over
Many of the statistical tests for the DTWF are falling over with exceptions being thrown - I had to put in a try/except into get them to run. @DomNelson, can you go through these please and delete any tests you think aren't relevant, and fix any that you think are?

### PR DESCRIPTION
**Pull request by DomNelson**
Removed largely redundant and long-running tests, and moved others into block run if `--extended` flag is set. Also added a few missing packages to `requirements/verification.txt`

DTWF tests now adjust growth rates so that populations do not shrink to zero individuals if coalescence is slow. Fixes #767.

### CODE REVIEW

File: requirements/verification.txt [hunk 1]
```diff
@@ -4,5 +4,8 @@ dendropy
 matplotlib
 numpy
 pandas
-scipy
+scipy==1.2 # Problems with v1.3 currently
```
Comment (jeromekelleher) [inline requirements/verification.txt]: What's the issue with scipy 1.3? Be nice to have something a bit more concrete here.

**Comment by jeromekelleher** [comment]
Ah, there's an issue with ``__tmp_max_time`` in here, so some are failing. Can you do a quick replace of ``__tmp_max_time`` with ``end_time`` as you're in there please @DomNelson?

**Comment by jeromekelleher** [comment]
I implemented ``end_time`` properly last week, which is why it's showing up now.

**Comment by DomNelson** [comment]
Ahh ok yep I'm on it

File: requirements/verification.txt (hunk 1 above)
Comment (DomNelson) [reply requirements/verification.txt]: Ya good point - the issue is [here](https://github.com/statsmodels/statsmodels/issues/5747), to do with statsmodels importing a deprecated module. I'll make this a bit more descriptive.

**Comment by DomNelson** [comment]
Ok I *think* I've got it straightened out, running one last time to make sure

**Comment by jeromekelleher** [comment]
Great, thanks @DomNelson. Ping me here when they've run and I'll merge. Should be easy to cut the full 0.7.1 release then.

**Comment by DomNelson** [comment]
Ok all set - final issue was that I'd set `pyplot.xlabel = "DTWF"` which clobbered later calls to `pyplot.xlabel()`. Full suite running fine now.

**Comment by jeromekelleher** [comment]
Thanks @DomNelson - I'll release 0.7.1 sometime over the weekend. I think there's just a minor issue about the DemographyDebugger to resolve and to write some release notes.

**Pull request merged**